javascript-modal/main.js

```diff
@@ -0,0 +1,27 @@
+var $modalButton = document.querySelector('.modal-button');
+$modalButton.addEventListener('click', modalVisibilityOn);
+
+var $modelElement = document.querySelector('.modal');
+
+var $noButton = document.querySelector('.no-button');
+$noButton.addEventListener('click', modalVisibilityOff);
+
+var $overlayBG = document.querySelector('#overlay');
+
+var visibilitOff = true;
+
+function modalVisibilityOn(event) {
+  if (visibilitOff) {
+    $modelElement.className += ' position-appear';
+    $overlayBG.className = 'overlay-bg';
+    visibilitOff = false;
+  }
+}
+
+function modalVisibilityOff(event) {
+  if (!visibilitOff) {
```

Comment:
Same deal here. If this button is clicked, you always want to do the same thing: hide the modal